scipy/stats/tests/test_multivariate.py

```diff
@@ -1976,6 +1976,17 @@ def test_rvs_broadcasting(self):
         rvs = rv.rvs(size=(1000, 2), random_state=123)
         assert_allclose(rvs.mean(0), rv.mean(), rtol=1e-2)
 
+    @pytest.mark.parametrize('m, n', (
+        ([0, 0, 20, 0, 0], 5), ([0, 0, 0, 0, 0], 0), ([0, 0], 0),
```

Comment:
You already have this case in your test matrix. Or you had another case in mind maybe?
```suggestion
        ([0, 0, 20, 0, 0], 5), ([0, 0, 0, 0, 0], 0),
```

Reply:
> Or you had another case in mind maybe?

I just wanted to make sure that small arrays with all 0s don't error out.

Reply:
Oh, I see that this test case is just a duplicate of the one below. Will remove it as you have suggested here. Thanks for noticing!